media/css/firefox/new/reggie-watts.scss

```diff
@@ -0,0 +1,289 @@
+            height: 107px;
+            margin: 0 auto;
+            width: 108px;
+        }
+    }
+
+    h2 {
+        @include font-size-level1;
+    }
+
+    .copy {
+        color: #fff;
+        padding: 0;
+        width: auto;
+
+        .download-button,
+        .help {
+            margin-top: 60px;
+        }
+
+        .help {
+            a:active,
+            a:focus,
+            a:hover,
+            a:link,
+            a:visited {
+                color: #fff;
+                text-decoration: underline;
+            }
+        }
+
+        .fx-privacy-link {
+            a:active,
+            a:focus,
+            a:hover,
+            a:link,
+            a:visited {
+                color: #fff;
+            }
+        }
+    }
+
+    @media #{$mq-phone-wide} {
+        background: #73320c url('/media/img/firefox/new/reggie-watts/orange-wave.png') bottom left no-repeat;
+        background: url('/media/img/firefox/new/reggie-watts/orange-wave.png') bottom left no-repeat,
+                    url('/media/img/firefox/new/reggie-watts/oval.png') top -40px left 40% no-repeat,
+                    linear-gradient(to right, #73320c 0%,#e98103 80%,#ec8306 100%),
+                    #73320c;
+    }
+
+    @media #{$mq-desktop} {
+        background: #73320c url('/media/img/firefox/new/reggie-watts/reggie-watts.jpg') top right 60% no-repeat;
+        background: url('/media/img/firefox/new/reggie-watts/orange-wave.png') bottom left no-repeat,
+                    url('/media/img/firefox/new/reggie-watts/oval.png') top -40px left 40% no-repeat,
+                    url('/media/img/firefox/new/reggie-watts/reggie-watts.jpg') top right 60% no-repeat,
+                    linear-gradient(to right, #73320c 0%,#e98103 80%,#ec8306 100%),
+                    #73320c;
+
+        h1 {
+            margin: 80px 0 120px -10px;
+
+            &:before {
+                display: inline-block;
+                margin: 0 0 -40px 0;
+            }
+        }
+
+        &.content-section {
+            padding: 0;
+        }
+
+        .content {
+            min-height: 700px;
+        }
+
+        .copy {
+            float: left;
+            width: 50%;
+            padding: 0 30px;
+
+            .logo {
+                margin: 60px 0 80px;
+            }
+        }
+
+    }
+}
+
+// always hide special scene 2 button (as it forces direct download)
+#download-button-wrapper-desktop {
+    height: 0;
+    visibility: hidden;
+    width: 0;
+}
+
+/* -------------------------------------------------------------------------- */
+// Detail content pillars
+
+.details {
+    background: #f1f1f1;
+
+    ul {
+        margin-bottom: 0;
+
+        li {
+            margin-bottom: 60px;
+            list-style-type: none;
+            width: 100%;
+
+            &:last-child {
+                margin-bottom: 0;
+            }
+        }
+
+        h3 {
+            margin-bottom: 20px;
+        }
+
+        p {
+            line-height: 30px;
```

Comment:
Should usually avoid fixed-unit line-heights. If you want to bump this up from the global setting `line-height: 1.66;` would get you 30px at an 18px font size.